model/model.h

```diff
@@ -0,0 +1,47 @@
+#ifndef MODEL_H
+#define MODEL_H
+
+#include <date.h>
+#include <chrono>
+#include <iostream>
+
+namespace model {
+
+class Message {
+private:
+    date::year_month_day message_date = date::year_month_day{
+        date::floor<date::days>(std::chrono::system_clock::now())};
+    date::hh_mm_ss<std::chrono::duration<long long, std::ratio<1, 1000000000>>>
+        message_time = date::make_time(
+            std::chrono::system_clock::now() -
+            date::floor<date::days>(std::chrono::system_clock::now()));
```

Comment:
На моем маке не собирается, потому что эта жуткая константа разная под разнымит системами.

```suggestion
    date::hh_mm_ss<std::chrono::system_clock::duration>
        message_time = date::make_time(
            std::chrono::system_clock::now() -
            date::floor<date::days>(std::chrono::system_clock::now()));
```